tests/rs_sqla_test_utils/models.py

```diff
@@ -14,6 +14,14 @@ class Basic(Base):
     )
 
 
+class BasicInSchema(Base):
+    __tablename__ = 'schema.basic'
+    col1 = sa.Column(
+        sa.Integer(), primary_key=True,
+        info={'distkey': True, 'sortkey': True}
+    )
```

Comment:
@berdario - Thanks for reporting this behavior and supplying the test case.

I'm taking a look now. I pulled down your branch and I'm going to try running it against our test cluster. I'll test out @graingert's suggestion for using __table_args__, too.